st2client/st2client/formatters/execution.py

```diff
@@ -21,6 +21,11 @@
 
 import yaml
 
+try:
+    from yaml import CSafeDumper as YamlSafeDumper
```

Comment:
Could we possibly add a comment to explain why we may not have CSafeDumper, and need this catch/exception?

Reply:
Funny enough, I just added similar change recently (a521bde4bd4f6ffb47410fff560e34763c39da04).

Technically as long as people are using official packages it should be there. But if they do from source install, etc it may not be.

Agreed it's a good ideal and will add a comment as well :)

Reply:
83627682d